pkgs/applications/version-management/sapling/default.nix

```diff
@@ -94,13 +128,19 @@ let
       curl
       pkg-config
     ] ++ (with rustPlatform; [
-      cargoSetupHook
+      myCargoSetupHook
       rust.cargo
       rust.rustc
     ]);
 
     buildInputs = [
+      curl
       openssl
+      libiconv
```

Comment:
Is `libiconv` really needed on Linux? IIRC it's only needed on Darwin

Reply:
Yes, we should move this into the darwin inputs.